src/components/DataInput/inputHandlerFactories.ts

```diff
@@ -9,7 +9,7 @@ type booleanSetter = {
     (checked: boolean): void;
 };
 
-type booleanGroup = {
+export type booleanGroup = {
```

Comment:
`booleanGroup` should ideally be `BooleanGroup`

Reply:
This is already in main and will be raise as a separate ticket (discussed with @TomCusack-Softwire). But agreed that this should be consistent throughout.